### PR TITLE
Reload code in node webserver

### PR DESCRIPTION
**Pull request by websoftwares**

### CODE REVIEW

**Comment by websoftwares** [comment]
@rodislav please CR

File: README.md [hunk 1]
```diff
@@ -11,5 +11,5 @@ $ docker build -t lr-restify .
 Running your image with -d runs the container in detached mode, leaving the container running in the background. The -p flag redirects a public port to a private port inside the container. Run the image you previously built:
 
 ```
-$ docker run -v $PWD:/usr/src/app -p 49160:8080 -d lr-restify
-```
\ No newline at end of file
+$ docker run -v $PWD/src/:/usr/src/app/src/ -p 49160:8080 -d lr-restify
```
Comment (rodislav) [inline README.md]: I think we could use `pwd` instead of $PWD, because the variable might not work everytime, can you try it ?

File: src/app.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 var restify = require('restify');
 
 function respond(req, res, next) {
-    res.send('get a     drink ' + req.params.name);
+    res.send('get a fucking small drink ' + req.params.name);
```
Comment (rodislav) [inline src/app.js]: big man! big drink!!!!!

File: README.md (hunk 1 above)
Comment (websoftwares) [inline README.md]: Can be merged?

**Pull request merged**